perun-base/src/main/java/cz/metacentrum/perun/core/api/AttributeDefinition.java

```diff
@@ -38,6 +38,12 @@
 	 */
 	private boolean writable;
 
+	/**
+	 * If the attribute values must be unique. For multivalued types like java.util.ArrayList, each value in the list
+	 * for a given object must be unique among all values for all objects.
+	 */
+	private boolean unique;
```

Comment:
It probably doesn't have to be part of this pull-request, but CLI a GUI must reflect new AttributeDefinition property. Please add it (also to print and create tools) or make a note and this can be merged only after change is done in separate pull-request.

Reply:
I have created issue for this - https://homeproj.cesnet.cz/issues/3948